src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx

```diff
@@ -0,0 +1,291 @@
+  measureStore: {
+    updateMeasure: jest.fn(),
+    state: jest.fn(),
+    initialState: jest.fn(),
+    subscribe: (set) => {
+      return { unsubscribe: () => null };
+    },
+  },
+  routeHandlerStore: {
+    subscribe: (set) => {
+      set();
+      return { unsubscribe: () => null };
+    },
+    updateRouteHandlerState: () => null,
+    state: { canTravel: true, pendingPath: "" },
+    initialState: { canTravel: true, pendingPath: "" },
+  },
+
+  checkUserCanEdit: jest.fn(() => {
+    return true;
+  }),
+}));
+
+const useMeasureServiceApiMock =
+  useMeasureServiceApi as jest.Mock<MeasureServiceApi>;
+
+let serviceApiMock: MeasureServiceApi;
+
+describe("MeasureInformation component", () => {
+  const { getByTestId, findByTestId, findAllByTestId, getByText } = screen;
+
+  test("Measure Group Scoring renders to correct options length, and defaults to empty string", async () => {
+    measureStore.state.mockImplementationOnce(() => measure);
+    checkUserCanEdit.mockImplementationOnce(() => true);
+
+    render(<BaseConfiguration />);
+
+    const scoringSelectInput = getByTestId(
+      "scoring-select-input"
+    ) as HTMLInputElement;
+    expect(scoringSelectInput.value).toBe("");
+
+    // options will be rendered only after clicking the select,
+    const scoringSelect = getByTestId("scoring-select");
+    const scoringSelectDropdown = within(scoringSelect).getByRole(
+      "button"
+    ) as HTMLInputElement;
+    userEvent.click(scoringSelectDropdown);
+
+    const optionsList = await findAllByTestId(/scoring-option/i);
+    expect(optionsList).toHaveLength(4);
+  });
+
+  test("Change of Measure Group Scoring enables Discard button and click Discard resets the form", async () => {
+    measureStore.state.mockImplementationOnce(() => measure);
+    checkUserCanEdit.mockImplementationOnce(() => true);
+
+    render(<BaseConfiguration />);
+
+    const scoringSelectInput = getByTestId(
+      "scoring-select-input"
+    ) as HTMLInputElement;
+
+    const scoringSelect = getByTestId("scoring-select");
+    const scoringSelectDropdown = within(scoringSelect).getByRole(
+      "button"
+    ) as HTMLInputElement;
+    userEvent.click(scoringSelectDropdown);
+
+    // Change the scoring value
+    fireEvent.change(scoringSelectInput, {
+      target: { value: "Cohort" },
+    });
+    expect(scoringSelectInput.value).toBe("Cohort");
+
+    const cancelButton = getByTestId("cancel-button");
+    expect(cancelButton).toBeInTheDocument();
+    await waitFor(() => expect(cancelButton).toBeEnabled());
+    act(() => {
+      fireEvent.click(cancelButton);
+    });
+
+    const discardDialog = await getByTestId("discard-dialog");
+    expect(discardDialog).toBeInTheDocument();
+    const continueButton = await getByTestId("discard-dialog-continue-button");
+    expect(continueButton).toBeInTheDocument();
+    fireEvent.click(continueButton);
+    // await waitFor(() => {
+    //   expect(scoringSelectInput.value).toBe("");
+    // });
```

Comment:
Is this commented out intentionally? This check may be required

Reply:
That was a mistake, I uncommented it out.